sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-servicebus/pom.xml

```diff
@@ -31,7 +31,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-messaging-servicebus</artifactId>
-      <version>7.0.0-beta.5</version> <!-- {x-version-update;unreleased_com.azure:azure-messaging-servicebus;dependency} -->
+      <version>7.0.0-beta.6</version> <!-- {x-version-update;unreleased_com.azure:azure-messaging-servicebus;dependency} -->
```

Comment:
This should be `7.0.0-beta.5(dependency version)`, not `7.0.0-beta.6(current version)`

Reply:
This was by design , no changes in `com.azure:azure-messaging-servicebus` should break azure-spring-boot-sample-servicebus .